mod-source-record-storage-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -93,6 +93,18 @@
           "fieldName": "id",
           "tOps": "ADD"
         }
+      ],
+      "foreignKeys": [
+        {
+          "fieldName": "rawRecordId",
+          "targetTable": "raw_records",
+          "tOps": "ADD"
+        },
+        {
+          "fieldName": "parsedRecordId",
+          "targetTable": "marc_records",
```

Comment:
parsedRecord can be any type, not just MARC, therefore they will be stored in different tables. It is incorrect to put a foreignKey constraint on this field referencing marc_records table

Reply:
Please talk it over with @TarasSpashchenko. If it's bad approach the you will able to revert last commit and use Btree indexes implementation of that I had already pushed before.

Reply:
@Anatolii-Starkov , we've already discussed on Friday, using you changes made before.

Reply:
Reverted last commit. We will stick with btrees for now.